pkg/cmd/local-csi-driver/driver.go

```diff
@@ -9,15 +9,15 @@ import (
 	"os"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/cmdutil"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/driver"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/driver/limit"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/driver/limit/xfs"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/driver/volume"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/genericclioptions"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/signals"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/util/fs"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/version"
+	"github.com/scylladb/local-csi-driver/driver"
```

Comment:
I forgot the `pkg/` part when pointing to the unterminated string
```suggestion
	"github.com/scylladb/local-csi-driver/pkg/driver"
```